src/meta/model_v2/src/object.rs

```diff
@@ -105,6 +105,14 @@ pub enum Relation {
         on_delete = "Cascade"
     )]
     SelfRef1,
+    #[sea_orm(
```

Comment:
I'm curious what's the difference between this new `Database2` and `Database` and why we can't get the correct table if we use `Database`?

Reply:
`Database` joins `object.oid` from `database.database_id`, but we actually requires join from `object.database_id` to `database.database_id`.